pyproject.toml

```diff
@@ -34,14 +37,22 @@ column_limit = 120
 max-line-length = 120
 disable = [
     "C0114", # Missing module docstring
+    "C0115", # Missing class docstring
     "C0116", # Missing function or method docstring
 ]
+good-names = ["ms"]
 
 [tool.setuptools]
 packages = [
-    "signwriting_transcription"
+    "signwriting_transcription",
+    "signwriting_transcription.pose_to_signwriting",
+    "signwriting_transcription.pose_to_signwriting.data",
+    "signwriting_transcription.pose_to_signwriting.joeynmt_pose",
 ]
 
 [tool.pytest.ini_options]
 addopts = "-v"
 testpaths = ["signwriting_transcription"]
+
+[project.scripts]
+signwriting_transcription = "signwriting_transcription.pose_to_signwriting.bin:main"
```

Comment:
i think the name of the cli tool should be `pose_to_signwriting`, to allow for `pose_to_vq_to_signwriting` in the future, or `video_to_signwriting` in the far future